website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -31,7 +31,11 @@ You will need a Jira admin user.
 
 ## Installation
 
-If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+Start by visiting the [Landing page for the Unleash Enterprise For Jira plugin on Atlassian marketplace](https://marketplace.atlassian.com/apps/1231447/unleash-enterprise-for-jira?hosting=cloud&tab=overview).
```

Comment:
Probably needs a mention that this is only for Pro and Enterprise, if you're on OSS this straight up won't work

Reply:
Ah, this is a very good point!

Reply:
Thank you, yes, very good point!